CAPS plugins fail to build with g++ 5

Using code pulled today:
```
plugins/LadspaEffect/caps/CMakeFiles/caps.dir/build.make:446: die Regel für Ziel „plugins/LadspaEffect/caps/CMakeFiles/caps.dir/SweepVF.cc.o“ scheiterte
make[2]: *** [plugins/LadspaEffect/caps/CMakeFiles/caps.dir/SweepVF.cc.o] Fehler 1
CMakeFiles/Makefile2:1869: die Regel für Ziel „plugins/LadspaEffect/caps/CMakeFiles/caps.dir/all“ scheiterte
make[1]: *** [plugins/LadspaEffect/caps/CMakeFiles/caps.dir/all] Fehler 2
make[1]: *** Auf noch nicht beendete Prozesse wird gewartet …
[  7%] Building C object plugins/LadspaEffect/swh/CMakeFiles/gsm.dir/gsm/short_term.c.o
[  7%] Building C object plugins/LadspaEffect/swh/CMakeFiles/gsm.dir/gsm/table.c.o
[  7%] Linking C static library libgsm.a
[  7%] Built target gsm
[  7%] Linking C shared module ../../ladspa/imp_1199.so
[  7%] Built target imp_1199
Makefile:138: die Regel für Ziel „all“ scheiterte
make: *** [all] Fehler 2
```

@tresf 
what is `LLVM/Clang` ?

Here the full log:

```shell
Already up-to-date.
removed 'CMakeCache.txt'
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
-- Found Git: /usr/bin/git (found version "2.7.4") 

Configuring LMMS
--------------------------
* Project version             : 1.2.0-rc4.36
*   Major version             : 1
*   Minor version             : 2
*   Release version           : 0
*   Stage version             : rc4
*   Build version             : 36
*

Optional Version Usage:
--------------------------
*   Override version:           -DFORCE_VERSION=x.x.x-x
*   Ignore Git information:     -DFORCE_VERSION=internal

PROCESSOR: x86_64
Machine: x86_64-linux-gnu
-- Target host is 64 bit
-- Looking for include file stdint.h
-- Looking for include file stdint.h - found
-- Looking for include file stdlib.h
-- Looking for include file stdlib.h - found
-- Looking for include file pthread.h
-- Looking for include file pthread.h - found
-- Looking for include file semaphore.h
-- Looking for include file semaphore.h - found
-- Looking for include file unistd.h
-- Looking for include file unistd.h - found
-- Looking for include file sys/types.h
-- Looking for include file sys/types.h - found
-- Looking for include file sys/ipc.h
-- Looking for include file sys/ipc.h - found
-- Looking for include file sys/shm.h
-- Looking for include file sys/shm.h - found
-- Looking for include file sys/time.h
-- Looking for include file sys/time.h - found
-- Looking for include file sched.h
-- Looking for include file sched.h - found
-- Looking for include file sys/soundcard.h
-- Looking for include file sys/soundcard.h - found
-- Looking for include file soundcard.h
-- Looking for include file soundcard.h - not found
-- Looking for include file fcntl.h
-- Looking for include file fcntl.h - found
-- Looking for include file sys/ioctl.h
-- Looking for include file sys/ioctl.h - found
-- Looking for include file ctype.h
-- Looking for include file ctype.h - found
-- Looking for include file string.h
-- Looking for include file string.h - found
-- Looking for include file process.h
-- Looking for include file process.h - not found
-- Looking for include file locale.h
-- Looking for include file locale.h - found
-- Looking for Q_WS_X11
-- Looking for Q_WS_X11 - found
-- Looking for Q_WS_WIN
-- Looking for Q_WS_WIN - not found
-- Looking for Q_WS_QWS
-- Looking for Q_WS_QWS - not found
-- Looking for Q_WS_MAC
-- Looking for Q_WS_MAC - not found
-- Found Qt4: /usr/bin/qmake (found suitable version "4.8.7", minimum required is "4.6.0;COMPONENTS;QtCore;QtGui;QtXml") 
-- Found Qt translations in /usr/share/qt4/translations
-- Checking for module 'sndfile>=1.0.11'
--   Found sndfile, version 1.0.25
-- Checking for module 'carla-standalone>=1.9.5'
--   Found carla-standalone, version 1.9.5
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found SDL: /usr/lib/x86_64-linux-gnu/libSDL.so;-lpthread (found version "1.2.15") 
-- Found STK: /usr/lib/x86_64-linux-gnu/libstk.so
-- Checking for module 'portaudio-2.0'
--   Found portaudio-2.0, version 19
-- Found Portaudio: portaudio;asound;m;pthread
-- Found SOUNDIO: /usr/lib/x86_64-linux-gnu/libsoundio.so  
-- Checking for module 'libpulse'
--   Found libpulse, version 8.0
-- Found PulseAudio Simple: /usr/lib/x86_64-linux-gnu/libpulse.so
-- Found Lame: /usr/include  
-- Looking for vorbis_bitrate_addblock in vorbis
-- Looking for vorbis_bitrate_addblock in vorbis - found
-- Found OggVorbis: /usr/lib/x86_64-linux-gnu/libogg.so;/usr/lib/x86_64-linux-gnu/libvorbis.so;/usr/lib/x86_64-linux-gnu/libvorbisfile.so;/usr/lib/x86_64-linux-gnu/libvorbisenc.so
-- Looking for snd_seq_create_simple_port in asound
-- Looking for snd_seq_create_simple_port in asound - found
-- Found ALSA: /usr/lib/x86_64-linux-gnu/libasound.so
-- Looking for include file machine/soundcard.h
-- Looking for include file machine/soundcard.h - not found
-- Looking for include file linux/awe_voice.h
-- Looking for include file linux/awe_voice.h - not found
-- Looking for include file awe_voice.h
-- Looking for include file awe_voice.h - not found
-- Looking for include file /usr/src/sys/i386/isa/sound/awe_voice.h
-- Looking for include file /usr/src/sys/i386/isa/sound/awe_voice.h - not found
-- Looking for include file /usr/src/sys/gnu/i386/isa/sound/awe_voice.h
-- Looking for include file /usr/src/sys/gnu/i386/isa/sound/awe_voice.h - not found
-- Looking for C++ include sys/asoundlib.h
-- Looking for C++ include sys/asoundlib.h - found
-- Looking for C++ include alsa/asoundlib.h
-- Looking for C++ include alsa/asoundlib.h - found
-- Looking for snd_pcm_resume in asound
-- Looking for snd_pcm_resume in asound - found
-- Checking for module 'jack>=0.77'
--   Found jack, version 1.9.11-RC1
-- Checking for module 'fftw3f>=3.0.0'
--   Found fftw3f, version 3.3.4
-- Found OpenGL: /usr/lib/x86_64-linux-gnu/libGL.so  
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found X11: /usr/lib/x86_64-linux-gnu/libX11.so
-- Found FLTK: fltk_images;fltk_forms;fltk_gl;/usr/lib/x86_64-linux-gnu/libGL.so;fltk  
-- Checking for module 'fluidsynth>=1.0.7'
--   Found fluidsynth, version 1.1.6
-- Checking for module 'gig'
--   Found gig, version 3.3.0
-- Could NOT find Wine (missing:  WINE_INCLUDE_DIRS) 
-- Checking for module 'samplerate>=0.1.8'
--   Found samplerate, version 0.1.8
-- Setting build type to 'Release' as none was specified.
-- Could NOT find Doxygen (missing:  DOXYGEN_EXECUTABLE) 

Installation Summary
--------------------
* Install Directory           : /usr/local

Supported audio interfaces
--------------------------
* ALSA                        : OK
* JACK                        : OK (weak linking enabled)
* OSS                         : OK
* Sndio                       : 
* PortAudio                   : OK
* libsoundio                  : OK
* PulseAudio                  : OK
* SDL                         : OK

Supported MIDI interfaces
-------------------------
* ALSA                        : OK
* OSS                         : OK
* Sndio                       : 
* JACK                        : OK (weak linking enabled)
* WinMM                       : <not supported on this platform>
* AppleMidi                   : <not supported on this platform>

Supported file formats for project export
-----------------------------------------
* WAVE                        : OK
* OGG/VORBIS                  : OK
* MP3/Lame                    : OK

Optional plugins
----------------
* Carla Patchbay & Rack       : OK
* SoundFont2 player           : OK
* Stk Mallets                 : OK
* VST-instrument hoster       : not found, please install (lib)wine-dev (or similar) - 64 bit systems additionally need gcc-multilib and g++-multilib
* VST-effect hoster           : not found, please install (lib)wine-dev (or similar) - 64 bit systems additionally need gcc-multilib and g++-multilib
* CALF LADSPA plugins         : OK
* CAPS LADSPA plugins         : OK
* CMT LADSPA plugins          : OK
* TAP LADSPA plugins          : OK
* SWH LADSPA plugins          : OK
* GIG player                  : OK


-----------------------------------------------------------------
IMPORTANT:
after installing missing packages, remove CMakeCache.txt before
running cmake again!
-----------------------------------------------------------------



-- Configuring done
-- Generating done
-- Build files have been written to: ./lmms/build
Scanning dependencies of target manpage
Scanning dependencies of target bin2res
[  0%] Built target manpage
[  0%] Automatic moc for target lmms
[  0%] Automatic moc for target lmmsobjs
[  0%] Built target bin2res
[  0%] Built target lmms_automoc
[  1%] Built target rpmalloc
[  1%] Generating moc_LadspaControlDialog.cxx
[  1%] Generating moc_ladspa_port_dialog.cxx
[  1%] Built target lmmsobjs_automoc
[  1%] Generating embedded_resources.h
[  1%] Generating embedded_resources.h
[  1%] Built target hydrogenimport
[  1%] Generating moc_ladspa_browser.cxx
[  2%] Generating moc_ladspa_description.cxx
[  2%] Generating moc_LadspaEffect.cxx
[  2%] Building CXX object plugins/LadspaEffect/caps/CMakeFiles/caps.dir/SweepVF.cc.o
[  2%] Building CXX object plugins/LadspaEffect/caps/CMakeFiles/caps.dir/VCO.cc.o
[  3%] Generating moc_LadspaControls.cxx
Scanning dependencies of target ladspabrowser
[  3%] Building CXX object plugins/ladspa_browser/CMakeFiles/ladspabrowser.dir/ladspa_browser.cpp.o
Scanning dependencies of target ladspaeffect
[  3%] Building CXX object plugins/LadspaEffect/CMakeFiles/ladspaeffect.dir/LadspaEffect.cpp.o
[  3%] Building CXX object plugins/LadspaEffect/CMakeFiles/ladspaeffect.dir/LadspaControls.cpp.o
In file included from ./lmms/plugins/LadspaEffect/caps/SweepVF.cc:33:0:
./lmms/plugins/LadspaEffect/caps/basics.h: In member function 'sample_t Plugin::getport_unclamped(int)':
./lmms/plugins/LadspaEffect/caps/basics.h:187:21: error: 'isinf' was not declared in this scope
     return (isinf (v) || isnan(v)) ? 0 : v;
                     ^
./lmms/plugins/LadspaEffect/caps/basics.h:187:21: note: suggested alternative:
In file included from /usr/include/c++/5/random:38:0,
                 from /usr/include/c++/5/bits/stl_algo.h:66,
                 from /usr/include/c++/5/algorithm:62,
                 from ./lmms/plugins/LadspaEffect/caps/SweepVF.cc:31:
/usr/include/c++/5/cmath:621:5: note:   'std::isinf'
     isinf(_Tp __x)
     ^
In file included from ./lmms/plugins/LadspaEffect/caps/SweepVF.cc:33:0:
./lmms/plugins/LadspaEffect/caps/basics.h:187:33: error: 'isnan' was not declared in this scope
     return (isinf (v) || isnan(v)) ? 0 : v;
                                 ^
./lmms/plugins/LadspaEffect/caps/basics.h:187:33: note: suggested alternative:
In file included from /usr/include/c++/5/random:38:0,
                 from /usr/include/c++/5/bits/stl_algo.h:66,
                 from /usr/include/c++/5/algorithm:62,
                 from ./lmms/plugins/LadspaEffect/caps/SweepVF.cc:31:
/usr/include/c++/5/cmath:641:5: note:   'std::isnan'
     isnan(_Tp __x)
     ^
plugins/LadspaEffect/caps/CMakeFiles/caps.dir/build.make:446: recipe for target 'plugins/LadspaEffect/caps/CMakeFiles/caps.dir/SweepVF.cc.o' failed
make[2]: *** [plugins/LadspaEffect/caps/CMakeFiles/caps.dir/SweepVF.cc.o] Error 1
CMakeFiles/Makefile2:1869: recipe for target 'plugins/LadspaEffect/caps/CMakeFiles/caps.dir/all' failed
make[1]: *** [plugins/LadspaEffect/caps/CMakeFiles/caps.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[  3%] Building CXX object plugins/LadspaEffect/CMakeFiles/ladspaeffect.dir/LadspaControlDialog.cpp.o
[  3%] Building CXX object plugins/ladspa_browser/CMakeFiles/ladspabrowser.dir/ladspa_description.cpp.o
[  3%] Building CXX object plugins/LadspaEffect/CMakeFiles/ladspaeffect.dir/LadspaSubPluginFeatures.cpp.o
[  3%] Building CXX object plugins/LadspaEffect/CMakeFiles/ladspaeffect.dir/moc_LadspaEffect.cxx.o
[  3%] Building CXX object plugins/LadspaEffect/CMakeFiles/ladspaeffect.dir/moc_LadspaControls.cxx.o
[  3%] Building CXX object plugins/ladspa_browser/CMakeFiles/ladspabrowser.dir/ladspa_port_dialog.cpp.o
[  3%] Building CXX object plugins/LadspaEffect/CMakeFiles/ladspaeffect.dir/moc_LadspaControlDialog.cxx.o
[  3%] Building CXX object plugins/ladspa_browser/CMakeFiles/ladspabrowser.dir/moc_ladspa_browser.cxx.o
[  3%] Building CXX object plugins/ladspa_browser/CMakeFiles/ladspabrowser.dir/moc_ladspa_description.cxx.o
[  3%] Linking CXX shared module ../libladspaeffect.so
[  3%] Building CXX object plugins/ladspa_browser/CMakeFiles/ladspabrowser.dir/moc_ladspa_port_dialog.cxx.o
[  3%] Built target ladspaeffect
[  3%] Linking CXX shared module ../libladspabrowser.so
[  3%] Built target ladspabrowser
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/plugins/LadspaEffect/caps/SweepVF.cc
+++ b/plugins/LadspaEffect/caps/SweepVF.cc
@@ -28,9 +28,9 @@
 	02111-1307, USA or point your web browser to http://www.gnu.org.
 */
 
-#include <algorithm>
-
 #include "basics.h"
+
+#include <algorithm>
 
 #include "SweepVF.h"
 #include "Descriptor.h"

